src/app/CommandHandler.h

```diff
@@ -171,6 +181,15 @@ class CommandHandler : public Messaging::ExchangeDelegate
     void OnInvokeCommandRequest(Messaging::ExchangeContext * ec, const PayloadHeader & payloadHeader,
                                 System::PacketBufferHandle && payload, bool isTimedInvoke);
 
+    /**
+     * Checks that all CommandDataIB within InvokeRequests is correct as per spec.
+     *
+     * This also build a lookup table that is multipurpose. It helps validate that all elements
+     * in the invoke request are unique, but also used when later on when populating
+     * elements of the InvokeResponse.
```

Comment:
Please remove the implementation details, and simply state that this builds the data structures to make sure all commands are responded.